packages/webcrypto-ed25519-polyfill/src/__tests__/install-test.ts

```diff
@@ -10,6 +10,20 @@ import {
 
 jest.mock('../secrets');
 
+const ED25519_ALGORITHM_IDENTIFIERS = [
+    'Ed25519',
+    'ED25519',
+    'eD25519',
+    { name: 'Ed25519' },
+    { name: 'ED25519' },
+    { name: 'eD25519' },
+];
```

Comment:
Shouldn't this include `ed25519` lowercase? Surely has to be more ordinary to call with that than `eD...`

Reply:
LOL. Totally missed that.